src/lib.rs

```diff
@@ -1,4 +1,8 @@
+#[macro_use] extern crate serde_derive;
```

Comment:
can this be more nicely done by putting in toml file perhaps, or can it at least be moved down?